### ISSUE TITLE
Conda recipe

### ISSUE THREAD

**Issue by sarthakpati** [securefederatedai/openfl#46]
**Is your feature request related to a problem? Please describe.**
Currently, OpenFL only has the option to be installed through pip, which prevents it to be added to packages that require C/C++ libraries.

**Describe the solution you'd like**
A conda recipe would be very useful to mitigate this, which I am happy to work on. 😄 

**Describe alternatives you've considered**
N.A.

**Additional context**
Needs #44

**Comment by dkurt** [comment]
Hi! In my experience, Conda config based project can install pip packages as well:
```yml
dependencies:
  - pip:
    - openfl
```

**Comment by sarthakpati** [comment]
True, but it makes it much easier when the packaging is sdist-based, rather than wheels.

**Comment by MasterSkepticista** [comment]
Closing due to inactivity.

**Comment by sarthakpati** [comment]
My bad - I can this on.

**Comment by sarthakpati** [comment]
Sorry, I completely forgot about this, so added it now: https://github.com/conda-forge/staged-recipes/pull/29540 

@MasterSkepticista please reopen this, and I can put in a PR once the recipe is available upstream.
